### PR TITLE
Minor rewrites

### PR DESCRIPTION
**Pull request by LightArrowsEXE**
To make it flow a bit better.

### CODE REVIEW

File: overview/roles.md [hunk 1]
```diff
@@ -161,20 +161,16 @@ own standards)
 Quality Checkers (abbreviated *QC*) are often the last eyes on an
 episode before it is released.
 They are responsible for inspecting the
-overall quality of every aspect of a show including:
-timing,
-typesetting,
-encode,
-and editing.
+ensuring the overall quality of the release is up to par
+with the group's standards.
```
Comment (FichteFoll) [inline overview/roles.md]: While I don't think we necessarily need to mention each role specifically, I do think we should clarify on the "standards" part and make a reference to all of the previous tasks as a whole.

Comment (LightArrowsEXE) [reply overview/roles.md]: Edited.

Comment (FichteFoll) [reply overview/roles.md]: Okay, `intricacies` is good for this.

However, could you reformat these sentences to use semantic linefeeds? They are explained in CONTRIBUTING.md.

Comment (LightArrowsEXE) [reply overview/roles.md]: Like so?

Comment (FichteFoll) [reply overview/roles.md]: Good enough.

**Pull request merged**